cmd/agent/install_script.sh

```diff
@@ -224,6 +228,10 @@ else
       no_start=true
     fi
   fi
+  if [ $site ]; then
```

Comment:
Why user `$site` and note directly `$DD_SITE` ?

Reply:
to improve a little bit the readability and maintainability of the script, all the supported env vars are listed once at the top of the script, and copied to script-local variables (and I think that's good practice)

Reply:
OK :+1: